src/components/binding/binding-loader.js

```diff
@@ -87,14 +92,14 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, fileName,
   };
   this.get(genotet.data.serverURL, params, function(data) {
     var track = {
+      gene: data.gene,
       fileName: fileName,
       overview: data,
       detail: data
     };
-    var addTrack = trackIndex == this.data.tracks.length;
     this.data.tracks[trackIndex] = track;
     this.updateRanges_();
-    if (addTrack) {
+    if (isAddTrack) {
       // Add one more track.
       this.loadBindingList();
```

Comment:
We may want to revisit how signals are fired for binding. 'track' used to mean a track is modified in the panel so that panel tracks can be properly updated [here](https://github.com/ViDA-NYU/genotet/blob/d1a7e9b554ca7355b731116099edf9b673107f1d/components/binding/binding.js#L94).

Since now we have 'addTrack', 'removeTrack' and so on, it's a bit confusing to use 'track' and maybe we should rename it to be more specific.